inlong-sdk/dataproxy-sdk/src/main/java/org/apache/inlong/sdk/dataproxy/config/HostInfo.java

```diff
@@ -50,9 +52,14 @@ public int compareTo(HostInfo other) {
         return referenceName.compareTo(other.getReferenceName());
     }
 
+    @Override
+    public int hashCode() {
```

Comment:
The `HostInfo` was used by Map, so it must implement both of `hashCode` and `equals` methods.